utils/web3client.py

```diff
@@ -340,6 +340,28 @@ def send_tokens(
         signed_tx = self.eth.account.sign_transaction(transaction, from_.key)
         tx = self.eth.send_raw_transaction(signed_tx.rawTransaction)
         return self.eth.wait_for_transaction_receipt(tx)
+    
+    @allure.step("Send all tokens from account")
+    def send_all_tokens_from_account(
+        self,
+        from_: eth_account.signers.local.LocalAccount,
+        to: tp.Union[str, eth_account.signers.local.LocalAccount],
+        value: int,
```

Comment:
maybe better to delete value and get it here with get balance function? "all tokens" in the name with value in parameters doesn't look logic

Reply:
fixed